exercise03/README.md

```diff
@@ -0,0 +1,3 @@
+## Exercise 3
+
+Given the tables and records defined `setup.sql`.  Write a query that returns all records and a count where `name` is not unique.
```

Comment:
So, do we not actually want the records themselves just the duplicates table (what you wrote above)? Or is the duplicates table in addition to pulling the records?

If first question is true, I'd write something like:

"Given the tables and records defined in `setup.sql`, write a query that returns all names that are not unique and the number of records associated with each of these non-unique names."

Then maybe put that little table you wrote so it's 100% clear.